server/apps/publish/formatters/newsml_1_2_formatter.py

```diff
@@ -241,15 +241,20 @@ def _format_place(self, article, descriptive_metadata):
         :param dict article:
         :param Element descriptive_metadata:
         """
-        for place in article.get('place', []):
-            if place.get('country') or place.get('state') or place.get('world_region'):
-                location = SubElement(descriptive_metadata, 'Location', attrib={'HowPresent': 'RelatesTo'})
-                if place.get('state'):
-                    SubElement(location, 'Property', {'FormalName': 'CountryArea'}).text = place.get('state')
-                if place.get('country'):
-                    SubElement(location, 'Property', {'FormalName': 'Country'}).text = place.get('country')
-                if place.get('world_region'):
-                    SubElement(location, 'Property', {'FormalName': 'WorldRegion'}).text = place.get('world_region')
+        if article.get('place'):
+            for place in article.get('place'):
```

Comment:
We can get rid of the outer `if` by changing this to `for place in article.get('place', {})`, and then dedenting the whole main block.